mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6147,6 +6145,21 @@ func containerClusterAutopilotCustomizeDiff(_ context.Context, d *schema.Resourc
 	return nil
 }
 
+// Default to VPC_NATIVE mode during initial creation
+func containerClusterNetworkingModeCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
```

Comment:
Initial creation default instead of a regular schema default.
I considered that container cluster was impactful enough to avoid a regular default breaking change.